source/administration/prev-config-settings.rst

```diff
@@ -0,0 +1,4424 @@
+Configuration Settings Prior to v5.12
```

Comment:
This one should be added to table of contents.

Reply:
👍 Added.